Database/Base/WorldBase.sql

```diff
@@ -1,397 +1,420 @@
-/*
-SQLyog Ultimate v12.4.2 (64 bit)
-MySQL - 10.2.6-MariaDB : Database - ace_world
-*********************************************************************
-*/
-
+CREATE DATABASE  IF NOT EXISTS `ace_world` /*!40100 DEFAULT CHARACTER SET utf8 */;
+USE `ace_world`;
+-- MySQL dump 10.13  Distrib 5.7.12, for Win64 (x86_64)
+--
+-- Host: localhost    Database: ace_world
+-- ------------------------------------------------------
+-- Server version	5.7.17-log
+
+/*!40101 SET @OLD_CHARACTER_SET_CLIENT=@@CHARACTER_SET_CLIENT */;
+/*!40101 SET @OLD_CHARACTER_SET_RESULTS=@@CHARACTER_SET_RESULTS */;
+/*!40101 SET @OLD_COLLATION_CONNECTION=@@COLLATION_CONNECTION */;
 /*!40101 SET NAMES utf8 */;
-
-/*!40101 SET SQL_MODE=''*/;
-
+/*!40103 SET @OLD_TIME_ZONE=@@TIME_ZONE */;
+/*!40103 SET TIME_ZONE='+00:00' */;
 /*!40014 SET @OLD_UNIQUE_CHECKS=@@UNIQUE_CHECKS, UNIQUE_CHECKS=0 */;
 /*!40014 SET @OLD_FOREIGN_KEY_CHECKS=@@FOREIGN_KEY_CHECKS, FOREIGN_KEY_CHECKS=0 */;
 /*!40101 SET @OLD_SQL_MODE=@@SQL_MODE, SQL_MODE='NO_AUTO_VALUE_ON_ZERO' */;
 /*!40111 SET @OLD_SQL_NOTES=@@SQL_NOTES, SQL_NOTES=0 */;
-CREATE DATABASE /*!32312 IF NOT EXISTS*/`ace_world` /*!40100 DEFAULT CHARACTER SET utf8 */;
-
-USE `ace_world`;
-
-/*Table structure for table `ace_weenie_class` */
-
-DROP TABLE IF EXISTS `ace_weenie_class`;
-
-CREATE TABLE `ace_weenie_class` (
-  `weenieClassId` INT(10) UNSIGNED NOT NULL,
-  `weenieClassDescription` TEXT NOT NULL,
-  PRIMARY KEY (`weenieClassId`),
-  UNIQUE KEY `idx_weenieName` (`weenieClassDescription`(100))
-) ENGINE=INNODB DEFAULT CHARSET=utf8;
-
 
-/*Table structure for table `ace_object` */
+--
+-- Table structure for table `ace_object`
+--
 
 DROP TABLE IF EXISTS `ace_object`;
-
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!40101 SET character_set_client = utf8 */;
 CREATE TABLE `ace_object` (
-  `aceObjectId` INT(10) UNSIGNED NOT NULL,
-  `aceObjectDescriptionFlags` INT(10) UNSIGNED NOT NULL,
-  `weenieClassId` INT(10) UNSIGNED NOT NULL,
-  `weenieHeaderFlags` INT(10) UNSIGNED NOT NULL,
-  `physicsDescriptionFlag` INT(10) UNSIGNED NOT NULL,
-  `currentMotionState` TEXT DEFAULT NULL,
+  `aceObjectId` int(10) unsigned NOT NULL,
+  `aceObjectDescriptionFlags` int(10) unsigned NOT NULL,
+  `weenieClassId` int(10) unsigned NOT NULL,
+  `weenieHeaderFlags` int(10) unsigned DEFAULT NULL,
+  `physicsDescriptionFlag` int(10) unsigned DEFAULT NULL,
+  `currentMotionState` text DEFAULT NULL,
   PRIMARY KEY (`aceObjectId`),
   KEY `idx_weenie` (`weenieClassId`),
   CONSTRAINT `fk_weenie_ao` FOREIGN KEY (`weenieClassId`) REFERENCES `ace_weenie_class` (`weenieClassId`)
```

Comment:
can't create a foreign key to ace_weenie_class before you create the table.  please test script in a blank DB and ensure it runs fully on the first attempt.